scripts/release.sh

```diff
@@ -421,8 +420,8 @@ function parse_flags() {
   if (( ! PUBLISH_RELEASE )); then
     (( has_gcr_flag )) && echo "Not publishing the release, GCR flag is ignored"
     (( has_gcs_flag )) && echo "Not publishing the release, GCS flag is ignored"
-    KO_DOCKER_REPO="ko.local"
-    KO_FLAGS="-L ${KO_FLAGS}"
+    KO_DOCKER_REPO="${KO_DOCKER_REPO:-ko.local}"
+    KO_FLAGS="${KO_FLAGS}"
```

Comment:
removed the -L because it's redundant with the ko.local value - and if set it would override the KO_DOCKER_REPO env var

Reply:
Makes sense. But now this line is unnecessary.

Reply:
doi :-)